definitions/features/foreman_install.rb

```diff
@@ -9,4 +9,16 @@ class Features::ForemanInstall < ForemanMaintain::Feature
       !feature(:instance).downstream && feature(:foreman_server)
     end
   end
+
+  def target_version
+    '3.11'
+  end
+
+  def current_version
+    @current_version ||= package_version(package_name)
+  end
+
+  def package_name
+    'foreman'
+  end
```

Comment:
this is already present in `foreman_server` feature, seems like duplication?
(and what about katello?)

Reply:
I don't know what the difference between the two features are and I didn't want to find out for this change :P